op-node/rollup/engine/params.go

```diff
@@ -0,0 +1,10 @@
+package engine
+
+import "time"
+
+const (
+	buildSealTimeout      = time.Second * 10
+	buildStartTimeout     = time.Second * 10
+	buildCancelTimeout    = time.Second * 10
+	payloadProcessTimeout = time.Second * 10
+)
```

Comment:
TODO: need to review these timeouts. They were set tight previously also, with the re-try engine-API call retrieving the actual work after it has been started by the previous call. But we may just  want to be more generous in timeouts.